packaging/opensuse/snapd.spec

```diff
@@ -27,22 +29,35 @@
 %{!?make_build: %global make_build %{__make} %{?_smp_mflags}}
```

Comment:
While at it, you can switch this to `%{?!..}`

Reply:
Yeah, I plan to make those consistent but in a separate cleanup PR.